File: Java/백준/Bronze/27160. 할리갈리/할리갈리.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;

        Map<String, Integer> map = new HashMap<>();
        map.put("STRAWBERRY", 0);
        map.put("BANANA", 0);
        map.put("LIME", 0);
        map.put("PLUM", 0);

        boolean bell = false;

        int N = Integer.parseInt(br.readLine());
        for (int i = 0; i < N; i++) {
            st = new StringTokenizer(br.readLine());
            String fruit = st.nextToken();
            map.put(fruit, map.get(fruit) + Integer.parseInt(st.nextToken()));
        }

        for (String key : map.keySet()) {
            if (map.get(key) == 5) {
                bell = true;
                break;
            }
        }

        System.out.println(bell ? "YES" : "NO");

    }
}

File: Python/백준/Bronze/27160. 할리갈리/할리갈리.py
import sys
input = sys.stdin.readline

d = {"STRAWBERRY":0, "BANANA":0, "LIME":0, "PLUM":0}
bell = False

N = int(input())

for i in range(N):
    fruit, num = map(str, input().split())
    d[fruit] += int(num)

for v in d.values():
    if v == 5:
        bell = True
        break

print("YES" if bell else "NO")